### DISCUSSION TITLE
server: a problem with getting some useless data files in server's directory

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1727] [Q&A]
### Description

i really have no idea wy this is happening, but may be you will know about that: in the directory where my teamtalk server files i selected it to be stored, sometimes, when i chek that folder, i see some data files that are not related to any actual files in my teamtalk server: and i'm skared to just simply remove them, i need to create another servis of my teamtalk server with its all files and backup, and go there and delete all my actual files from the teamtalk, and then remove the remaning files from this folder and from the main folder too: this bug is bad because my server doesn't have so much space and these files are filling the space and they are not useful because they are not related to any files in my teamtalk server: but i don't know when this is happening, so i can't really reproduce for you: please take a look: thank you.

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by kirill-jjj** [comment]
i am not a developer, but you can safely remove them, and tmp files two. These are files that were on the server, but were not deleted for some reason. Maybe @CoBC know's an ancer?

**Reply by CoBC** [reply]
I checked on my server and I just see files data_x.dat created when I upload a file, but this file is deleted when I delete file. So, I don't really know of what files it's question here.

**Reply by kirill-jjj** [reply]
as I understand it, sometimes they are not deleted

**Comment by bear101** [comment]
You can look in tt5srv.xml to find the mapping to data files.

**Comment by amirmahdifard** [comment]
yes i know that, but there some files that the xml file does not mention them

**Comment by amirmahdifard** [comment]
sometimes, even if you delete all files in your all channels, stil some data files will remaning in server directory: this should be clear enough

**Comment by lbk2907** [comment]
I also have experience this kind of situation before.
